kolibri/deployment/default/settings/base.py

```diff
@@ -47,8 +47,10 @@
     'kolibri.auth.apps.KolibriAuthConfig',
     'kolibri.content',
     'kolibri.logger',
+    'kolibri.tasks.apps.KolibriTasksConfig',
```

Comment:
I haven't seen a classname referenced in INSTALLED_APPS before -- what's all this then?